src/simularium/VisData.ts

```diff
@@ -194,10 +113,29 @@ class VisData {
         this.lockedForFrame = true;
     }
 
+    public setCacheSettings(options: {
+        cacheEnabled?: boolean;
+        maxCacheSize?: number;
+    }): void {
+        const { cacheEnabled, maxCacheSize } = options;
+        if (cacheEnabled !== undefined) {
+            this.frameCache.changeSettings({ cacheEnabled: cacheEnabled });
+        }
+        if (maxCacheSize === undefined) {
+            return;
+        }
+        if (maxCacheSize < 0) {
+            this.frameCache.changeSettings({ maxSize: -1 });
+            return;
+        }
+        // cache must have at least one frame
+        const newCacheSize = maxCacheSize > 0 ? maxCacheSize : 1;
+        this.frameCache.changeSettings({ maxSize: newCacheSize });
+    }
```

Comment:
Making this change!

Possible that I did this because this degree of chaining was starting to make me uncomfortable:
`this.props.simulariumController.visData.frameCache.changeSettings`